### ISSUE TITLE
WG: YouTube Channel (Content, Curation, Production, Process)

### ISSUE THREAD

**Issue by halcyondude** [cncf/tag-observability#49]

**Comment by jsalinas29** [comment]
Hi team - I am happy to work with others on putting a process together here. Let me know if you're interested in pursuing this.

**Comment by halcyondude** [comment]
Yes indeed!  I'm adding bits now to add collaborators to this repo so we can assign issues.  From our discussion in the TAG meeting it sounds like you have a lot of great experience and ideas here!

**Comment by henrikrexed** [comment]
Hi, As discussed i am super motivated to build content and promote all the initiatives of CNCF.

**Comment by halcyondude** [comment]
I realize it's late notice, @henrikrexed @jsalinas29 could we find a time this week to connect & sync on logistics and next steps?  I would like to help facilitate this activity in whatever way(s) that I can.

Thanks!

**Comment by henrikrexed** [comment]
Hi,

I'm happy to launch this initiative, but we would need to understand how we would be able to publish the video on the CNCF channel. Or do you plan to create a Tag observability channel?
I have a colleague from Grafana who would be happy to assist me in this initiative.
let's sync on this to bring this initiative live.

**Comment by henrikrexed** [comment]
I would be back in France on the 20th of November, let's sync from the 21st

**Comment by alolita** [comment]
Thanks @henrikrexed There is a TAG observability channel where we post meeting recordings - lets chat on Slack.

**Comment by halcyondude** [comment]
I've brought this to TAG Contributor Strategy per our convo at most recent TAG Meeting.  @henrikrexed @alolita FYI

https://docs.google.com/document/d/1QKRyr5CNXx5Uc1nzMlyTq-gRKCIrBiYWGsiRZu_Dmyc/edit#heading=h.qpqb9wipdffm

**Comment by henrikrexed** [comment]
Hi,
As discussed during our last TAG meeting, the YouTube channel should cover content helping our community but also any CNCF observability projects in receiving ideas and feedback.
To achieve the objective, we should provide the following type of content:
- Ask Me Anything live with a maintainer of a given project. this format will share a quick demo/explanation and then mainly focus on answering questions from our community
- Observability solution webinar: Create specific videos with projects. The format of the video needs to be short and needs to provide enough details to help our community get started on the project
- Methodology webinar: specific content aimed to share best practices around observability. we could imagine having a specific episode on the Observability whitepapper.

This is just a short list shared during our meeting. But if you have any other ideas or needs, feel free to add your comments.
